Buddies/Onboarding/SignUpInfoVC.swift

```diff
@@ -188,7 +190,7 @@ class SignUpInfoVC: LoginBase, UIImagePickerControllerDelegate, UINavigationCont
             let blockedActivities: [String] = []
             let blockedBy: [String] = []
             let dateJoined = Timestamp(date: Date())
-            let loc = GeoPoint.init(latitude: 10, longitude: 10)
+            let loc = GeoPoint(latitude: 0, longitude: 0)//this should get replaced momentatily
```

Comment:
why even initialize it to 0, 0? If the user denies location, shouldn't we store "null" or something more clear like that?

Reply:
Just a spelling nit-pick spelling: "momentarily" ;)